website/docs/r/service.html.markdown

```diff
@@ -87,7 +87,7 @@ The block contains the following arguments:
   * `start_time` - The support hours' starting time of day.
   * `end_time` - The support hours' ending time of day.
 
-When using `type = "use_support_hours"` in `incident_urgency_rule` you must specify at least one (otherwise optional) `scheduled_actions` block.
+When using `type = "use_support_hours"` in `incident_urgency_rule` you can specify an optional `scheduled_actions` block.
```

Comment:
Based on the information I received from PagerDuty Engineering, the new language should be something more like, 

When using `type = "use_support_hours"` in `incident_urgency_rule` a `scheduled_actions` block is required.

Reply:
Thank you @stmcallister ! Reflected your comment.